### PR TITLE
chore: Isolate global constants to module.

### PR DESCRIPTION
**Pull request by SpcFORK**

### CODE REVIEW

**Comment by niceEli** [comment]
.gitignore is conflicting. removing .replit will fix it as .replit and replit.nix is now in gitignore.

the test failing seems to be my fault, will be fixed.

File: src/kaboom.ts [hunk 1]
```diff
@@ -369,11 +288,7 @@ export default (gopt: KaboomOpt = {}): KaboomCtx => {
     }
 
     // create a <canvas> if user didn't provide one
-    const canvas = gopt.canvas ?? (() => {
-        const canvas = document.createElement("canvas");
-        root.appendChild(canvas);
-        return canvas;
-    })();
+    const canvas = gopt.canvas ?? root.appendChild(document.createElement("canvas"))
```
Comment (lajbel) [inline src/kaboom.ts:291]: Not related to PR, I'll let it go this time

**Review by lajbel — APPROVED** [review]

**Comment by niceEli** [comment]
just updated tests. should fix the issue after pull

File: src/kaboom.ts (hunk 1 above)
Comment (SpcFORK) [reply src/kaboom.ts:291]: Yeah, that's fine, commited it and am not sure how to separate to different PR after.

**Pull request merged**